timetracker/timetracker.py

```diff
@@ -155,7 +155,7 @@ def hours_as_table(content, current_month, full, show_weekday):
     if full:
         column_headers = ["Date", "Hours", "Project", "Assigment Type", "Description"]
     else:
-        column_headers = ["Date", "Description"]
+        column_headers = ["Date", "Hours", "Project", "Task Category", "Task", "Description"]
```

Comment:
@v-romana shouldn't these fields be added only when `full` is `True`? The default behavior is showing only most meaningful fields.

Reply:
Ok after checking https://github.com/eyscode/timetracker/issues/15 I understand some stuff changed on web time tracker, I'm not sure if this PR covers all those changes, even if so the new fields should be added only when `full` is true not as the default behaviour.